README.md

```diff
@@ -36,12 +41,20 @@ Develop branch status
 
 4. pip install this package to your virtualenv
 
-        pip install data_api
+        pip install ../data_api
 
 5. set your `KB_AUTH_TOKEN` to your token string. In a KBase environment use `kbase-login` to retrieve a token, and check your `.kbase_config`. Once you have the value, run this command in the bash shell:
 
         export KB_AUTH_TOKEN="<value-you-grabbed>"
 
+6. Install example data
```

Comment:
Let's add something like (not sure this will come out in the comments):

See [Test data](README.md#test-data)